sdk/servicebus/azure-messaging-servicebus/src/main/java/com/azure/messaging/servicebus/implementation/models/QueueDescriptionEntryContent.java

```diff
@@ -52,18 +52,18 @@ public QueueDescriptionEntryContent setType(String type) {
      *
      * @return the queueDescription value.
      */
-    public QueueDescription getQueueDescription() {
-        return this.queueDescription;
+    public QueueProperties getQueueProperties() {
+        return this.queueProperties;
     }
 
     /**
      * Set the queueDescription property: Description of a Service Bus queue resource.
```

Comment:
Change this to `queueProperties`.

Reply:
These are auto-generated from AutoRest. In another PR, I'm going to pull the actual QueueDescription class into implementation and have a separate QueueProperties class.